From ecfdd45e5ba24adac4153b71c59f5bc8cdd226db Mon Sep 17 00:00:00 2001
From: sbwml <984419930@qq.com>
Date: Mon, 20 Mar 2023 05:58:09 +0800
Subject: [PATCH 10/11] rockchip: rk3568: enable usb2 support

---
 include/configs/rk3568_common.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/rk3568_common.h b/include/configs/rk3568_common.h
index d72840861c..9da67dc922 100644
--- a/include/configs/rk3568_common.h
+++ b/include/configs/rk3568_common.h
@@ -29,6 +29,9 @@
 #define SDRAM_UPPER_ADDR_MIN		0x100000000
 #endif
 
+#define CONFIG_USB_OHCI_NEW
+#define CONFIG_SYS_USB_OHCI_MAX_ROOT_PORTS	2
+
 #ifndef CONFIG_SPL_BUILD
 #define ENV_MEM_LAYOUT_SETTINGS		\
 	"scriptaddr=0x00c00000\0"	\
-- 
2.34.1

